### PR TITLE
feat: Modals

### PR DESCRIPTION
**Pull request by SuyashTiwariCoder**
Added:
1) Simple Modal
2) Stacked Modal

### CODE REVIEW

File: stylesheets/color.css [hunk 1]
```diff
@@ -59,7 +59,8 @@
     --white:white;
     --grey:grey;
     --voilet:blueviolet;
-
+    --teal:teal;
+
```
Comment (harshitadaswani) [inline stylesheets/color.css]: use hex, rgb or hsl format

**Review by harshitadaswani — APPROVED** [review]
Use color variables in either RGB, HSL or hex format

File: components/modals/modal.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+.modal--heading{
+    padding: 2rem;
+}
+
+.modal--text{
+padding: 1rem;
+}
+
+.modal--stacked__btn{
+
```
Comment (dev-enforced) [inline components/modals/modal.css]: remove space.

**Review by dev-enforced — APPROVED** [review]
The code looks good . Suggested some minor changes . Refactor according to it and then merge

**Pull request merged**